COORDINATE_SYSTEM2/COORDINATE_SYSTEM2/PointsDistance.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace COORDINATE_SYSTEM2
+{
+    internal class PointsDistance //клас, в якому я маю конструктор, який збирає докупи інформацію про дві точки і відстань між ними
+    {
+        public Point Point1 { get; set; }
+        public Point Point2 { get; set; }
+        public double Distance { get; set; }
```

Comment:
you can make setters private if you have constructor, this way you can make sure that this value will be assigned only on instance creation

Reply:
I removed setter at all...